### ISSUE TITLE
Doesn't seem to work with Python "Anonymous Files"

### ISSUE THREAD

**Issue by SonOfDiablo** [MicrosoftDocs/intellicode#258]
Issue Type: <b>Bug</b>

Open new tab > Change language to Python > Write code...

Example code I used as a test:
```
x = 's'
x.
```

Suggestions I got was:
datetime
PyZipFile
ZipFile

![ym66R7LcGA](https://user-images.githubusercontent.com/9123250/102513161-27f60b00-408b-11eb-8b83-505bf7773454.png)

Extension version: 1.2.10
VS Code version: Code 1.52.1 (ea3859d4ba2f3e577a159bc91e3074c5d85c0523, 2020-12-16T16:34:46.910Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz (12 x 2592)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.84GB (4.24GB free)|
|Process Argv|--crash-reporter-id 890fbd32-b85c-48d5-894b-184331acf6c0|
|Screen Reader|no|
|VM|80%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
openlogontheside:30221877
python763:30178808
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
python504:30227505
vswsl492cf:30211402
wsl2prompt:30224612
pythonvsdeb440:30237827
unusedpromptcf:30224611
folderexplorercf:30224615
openfilemenu:30224647
pythonvsded773cf:30236630
vsjup459:30237187

```

</details>

**Comment by SonOfDiablo** [comment]
A quick update:

I tried saving the file and then it suddenly worked.
![9ZnCcBUYBC](https://user-images.githubusercontent.com/9123250/102513773-ee71cf80-408b-11eb-8504-9cca5a168f22.png)

**Comment by catycaldwell** [comment]
@savannahostrowski @jakebailey - adding to bubble up this feature request to the Python Language Server team.

**Comment by catycaldwell** [comment]
@SonOfDiablo, Thank you for providing this feedback and providing the work around. The IntelliCode team is looking into this. Thanks!

**Comment by jakebailey** [comment]
This is microsoft/pylance-release#739, and IC would be fixed with it.

**Comment by markw-t** [comment]
doc-bug to document current behavior as described by this bug.

**Comment by savannahostrowski** [comment]
This was fixed in Pylance 2021.1.0 and should no longer be an issue.

**Comment by markw-t** [comment]
Based on @savannahostrowski comment, this should no longer need documenting as it's fixed. Closing
